graphjet-core/src/test/java/com/twitter/graphjet/hashing/ArrayBasedIntToIntArrayMapTest.java

```diff
@@ -176,4 +179,26 @@ public void testRandomConcurrentReadWrites() {
       random
     );
   }
-}
+
+  @Test
+  public void testFailsToCreateArrayBasedIntToIntArrayMapTakingFourArgumentsThrowsIllegalArgumentException() {
```

Comment:
the name of the test suggests taking four arguments, but the actual test is for a negative number of nodes? a couple of other test cases with the name "four arguments" seem to have the similar mismatch.